lib/rules/shorthand-property-no-redundant-values/index.mjs

```diff
@@ -172,10 +173,15 @@ const rule = (primary) => {
 				decl.value = decl.value.replace(value, shortestFormString);
 			};
 
+			const index = declarationValueIndex(decl);
+			const endIndex = index + decl.value.length;
+
 			report({
 				message: messages.rejected,
 				messageArgs: [value, shortestFormString],
 				node: decl,
+				index,
+				endIndex,
```

Comment:
Same question here.

Is the issue you are trying to solve that there is a shared prefix both before and after the fix?

In this case the value and property are tightly coupled.
There is nothing wrong with writing `10px 10px` when looked at in isolation. The property name adds crucial context.

Reply:
> If we want to align replacement text with the ranges we can also emit the entire declaration as the replacement text.

We could but that would be less precise / too large.
i.e. we will want the range to be as small as possible

> The property name adds crucial context.

The formatters already give enough context.
i.e. range and rule
I think that reporting the value is better and would be consistent with the current message arguments.